main_routers/cookies_login_router.py

```
@@ -0,0 +1,229 @@
+        platforms = login_manager.get_supported_platforms()
+        return {
+            "success": True,
+            "data": {
+                p: {
+                    "name": info["name"],
+                    "methods": info["methods"],
+                    "default_method": info["default_method"]
+                } for p, info in platforms.items()
+            }
+        }
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)